packages/dev/core/src/Morph/morphTarget.ts

```diff
@@ -144,6 +144,23 @@ export class MorphTarget implements IAnimatable {
         return !!this._uv2s;
     }
 
+    /**
+     * Gets the number of vertices stored in this target
+     */
+    public get vertexCount(): number {

```

Comment:
This hurts :-) 

If we use it often let s cache the results ?

Reply:
This isn't so bad, because in 99.9% of cases, we'll have positions, so we'll only have one check.

What's more, the getter is not often used, only when the texture needs to be recreated: in 99.9% of cases, the texture will be created once, at start.